cisagov/crossfeed
# 1851 modify syncdb to allow for the population of structured content

## 🗣 Description ##

Updated syncdb to create some organizations have the parentId field populated in the database and those organizations are other organizations in the database to simulate users creating suborganizations.  

## 💭 Motivation and context ##

This closes 1851 and was needed to test the new organization, suborganization

## 🧪 Testing ##

Manual testing: 
Start the backend and the db. Then run the npm run syncdb -- -d populate and ensure that some of the organizations have the parentId populated.

Run crossfeed locally and make sure everything still loads and works on the organizations screen.

## ✅ Pre-approval checklist ##

      in code comments.
      to reflect the changes in this PR.

## ✅ Pre-merge checklist ##


## ✅ Post-merge checklist ##
